src/providers/postgres/qgspostgresdataitemguiprovider.cpp

```diff
@@ -149,6 +149,31 @@ void QgsPostgresDataItemGuiProvider::populateContextMenu( QgsDataItem *item, QMe
       maintainMenu->addAction( actionRefreshMaterializedView );
     }
     menu->addMenu( maintainMenu );
+
+    if ( layerInfo.relKind != Qgis::PostgresRelKind::View && layerInfo.relKind != Qgis::PostgresRelKind::MaterializedView )
+    {
+      QAction *actionMoveLayerToSchema = new QAction( tr( "Move %1 To Schema…" ).arg( typeName ), menu );
+      connect( actionMoveLayerToSchema, &QAction::triggered, this, [layerItem, context] { moveLayerToSchema( layerItem, context ); } );
+
+      // this action should sit in the Manage menu. If one does not exist, create it now
+      bool foundExistingManageMenu = false;
```

Comment:
How can there be an existing Manage menu?

Reply:
There is one created in `QgsLayerItemGuiProvider` which adds delete action. Even that one checks for existance of the  Manage menu. Wanted to follow the same logic ... not sure if we can rely on the order of those functions being called.